espanol/noche/noche.mdown

```diff
@@ -0,0 +1 @@
+Y en la noche se caso con su mujer y tomaron una botella de ron.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.